subsys/bluetooth/common/Kconfig

```diff
@@ -77,10 +77,21 @@ config BT_BUF_ACL_RX_SIZE
 	  In a Controller only build this will determine the maximum ACL size
 	  that the Controller will send to the Host.
 
+config BT_BUF_ACL_RX_COUNT_EXTRA
+	int "Number of extra incoming ACL data buffers"
+	default 0
+	range 0 65535
```

Comment:
Should this depend on `BT_CONN`?

Reply:
I am not sure, `CONFIG_BT_BUF_ACL_RX_COUNT` wasn't. Do you think it would be better?

Reply:
In some ways, yes. 

If BT_CONN=n, then this option has no purpose. Adding the dependency makes the user aware of that. 

On the other hand, if this value is used/referenced in header files that may be included when BT_CONN=n, then it will either give compile errors or require a guard. 

We have a dependency on the ISO buffers, which is why we need to do something like 

```
#if defined(CONFIG_BT_ISO)
#define BT_BUF_ISO_RX_SIZE BT_BUF_ISO_SIZE(CONFIG_BT_ISO_RX_MTU)
#define BT_BUF_ISO_RX_COUNT CONFIG_BT_ISO_RX_BUF_COUNT
#else
#define BT_BUF_ISO_RX_SIZE 0
#define BT_BUF_ISO_RX_COUNT 0
#endif /* CONFIG_BT_ISO */
```

Unfortunately, AFAIK, we don't have a `if not defined then 0 else CONFIG_X` macro for non-bool values